### ISSUE TITLE
Specify CMake generator possible?

### ISSUE THREAD

**Issue by dwang-hingehealth** [tomtzook/gradle-cmake#1]
Hi,

Is there a way to pass an arbitrary command line argument to CMake? What I would like to do is something like `cmake -G Xcode -DCMAKE_TOOLCHAIN_FILE=path/to/toolchain.cmake ..`. I'can't find how to pass ` -G Xcode` along.

Thanks!

**Comment by tomtzook** [comment]
At the moment, no. But it should be a simple change to allow support.
Will update as soon as I can.

**Comment by tomtzook** [comment]
Added feature in #3

**Comment by tomtzook** [comment]
Published version `1.1.0`.

Use

```groovy
cmake {
    targets {
        example_project {
            cmakeLists.set(file('src/CMakeLists.txt'))
            targetMachines.add(machines.host)
            generator.set("Xcode")
        }
    }
}
```
